### DISCUSSION TITLE
Save information from one of two fields in one column

### DISCUSSION THREAD

**Discussion by cyber-kivi** [Budibase/budibase#6154] [Help]
Hello there!
How can I make this:

1. The form has a text field for entering data into table1 (main) and also a line for selecting data from table2 with the option picker function;
2. Enter text or select from a list (table2);
3. Data from both fields saves in table1 in one column.

I found this https://docs.budibase.com/docs/referencing
There are two columns for saving data from two tables in one, I need one (in main table1) column for saving data from two fields (table1 text field and table2 list)

**Comment by melohagan** [comment]
Hi @cyber-kivi 

I believe you would just use a binding for this that combines the two field values in whichever format you like.

Something along the lines of: `Text field value is {{ Form.Fields.TextField }} and the picker value is {{ Form.Fields.Picker }}`

You can see the available bindings in the scrollable left hand pane.

More here: https://docs.budibase.com/docs/bindings
